### PR TITLE
Update new user and trial initiation email content

### PR DESCRIPTION
**Pull request by mark-youssef-bitwarden**
## 🎟️ Tracking

Asana
- https://app.asana.com/0/1205225580489126/1207892529921039/f

## 📔 Objective

This updates the email content for new users 

## 📸 Screenshots

mail google com_mail_u_0_ (4)

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Comment by mark-youssef-bitwarden** [comment]
> @mark-youssef-bitwarden We use sentence case for button labels throughout the product and transactional emails, could you update that in this PR? Thanks!

Thanks for the feedback @danielleflinn! Just to double-check my understanding, "Install the Extension" should read "Install the extension" correct?

**Comment by danielleflinn** [comment]
@mark-youssef-bitwarden yes exactly! Thank you

**Comment by mark-youssef-bitwarden** [comment]
@danielleflinn Here is the latest preview after feedback from marketing:
- Update button label to sentence-case
- Add divider between steps and extra links
- Update border radius of buttons and bottom box to be consistent with the marketing site style
- Shorten secrets manager copy

Let me know if the changes are all good with you

codepen io_Mark-Youssef_full_WNqoRjB (5)

**Comment by danielleflinn** [comment]
@mark-youssef-bitwarden LGTM thanks!

**Review by jlf0dev — APPROVED** [review]

File: src/Core/MailTemplates/Handlebars/TrialInitiation.text.hbs [hunk 1]
```diff
@@ -1,39 +1,50 @@
...
 
 
-    Download Options
-    ============
+2. Add passwords to your vault
+============
 
-    http://www.bitwarden.com/download
+Your secure vault is ready to safely store your sensitive information. Add logins, credit cards, notes, and identities for fast access and autofilling!
+Add passwords to your vault ({{WebVaultUrl}}/?utm_source=welcome_email&utm_medium=email)
```
Comment (Thomas-Avery) [inline src/Core/MailTemplates/Handlebars/TrialInitiation.text.hbs:18]:
```suggestion
Add passwords to your vault ({{{WebVaultUrl}}}/?utm_source=welcome_email&utm_medium=email)
```

:pick: 
I would also triple escape in the `text.hbs` templates to stay consistent with the `html.hbs` files.

There are a few occurrences of this.

I did notice it was like this in the old version of these `text.hbs` templates, so up to you.
 

Triple stash `{{{`tells handlebars not to HTML-escape the input.

https://handlebarsjs.com/guide/expressions.html#html-escaping

Typically, we use `{{` for everything and only use triple for generated URLs, so everything is properly HTML escaped.

**Review by Thomas-Avery — APPROVED** [review]
LGTM, an optional thing I noticed.

File: src/Core/MailTemplates/Handlebars/TrialInitiation.text.hbs (hunk 1 above)
Comment (mark-youssef-bitwarden) [reply src/Core/MailTemplates/Handlebars/TrialInitiation.text.hbs]: @Thomas-Avery Thanks! I updated it to `{{{`

**Review by Thomas-Avery — APPROVED** [review]

**Pull request merged**